cypress/fixtures/GDls.json

```diff
@@ -0,0 +1,12 @@
+{
+  "GD_creds": "{\"signature\":\"0xaaf82b85764cae823a3d01a635278c8702efab2c46ee4e64c7fd7850814ff1994a47049c7c616d7f27a820a1ee9163b61478a0025a25e5925af56908cfbae9af1c\",\"gdSignature\":\"0xcceabdc8203d57d3647b5811e90b9b407a78483755537addb126eb2efebdaf0e18a9c3fc635ec6d2c6cec4dc2699d19aef2f6270e34156b2e31ec1a56cdc21401c\",\"profilePublickey\":\"aUh1A1Ca5YOJhMbynKK9DPFhGW9ixqUuOtM9yEFaHuo.NbVqh0-cFSnNh5IzF8L4ANJ2lg-M6korLzd_i6B-_dk\",\"profileSignature\":\"SEA{\\\"m\\\":\\\"Login to GoodDAPP0d6f36772c0ade118400\\\",\\\"s\\\":\\\"kMAALrQ5ZjGUgX8e6fX6vgpeHZO7lZskKW2ybufO06HvBfQ6XmAwEXNp656T34gJgM2lkhSNvY5YZRLQ0khXnw==\\\"}\",\"nonce\":\"0d6f36772c0ade118400\",\"networkId\":122,\"jwt\":\"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJsb2dnZWRJbkFzIjoiMHhmNGFkYjJmOWE2ODYzNmZmMzYxZDZjOTgzMzQ3M2ZmYmI5NTJiNzVhIiwiZ2RBZGRyZXNzIjoiMHhkYzAwNDZiNzEzYjZlZDg3ZDRhYmE0NWZkMmJhMGVmZGFlNzk2ZmJlIiwicHJvZmlsZVB1YmxpY2tleSI6ImFVaDFBMUNhNVlPSmhNYnluS0s5RFBGaEdXOWl4cVV1T3RNOXlFRmFIdW8uTmJWcWgwLWNGU25OaDVJekY4TDRBTkoybGctTTZrb3JMemRfaTZCLV9kayIsImlhdCI6MTU5MDA3MzI2NH0.u8Z2_9YDq5pe1DrmZ5joXw9KCB3IyuYZAgUrX9QaUOI\"}",
```

Comment:
are those secrets from services used or all they are used as fixtures (e.g. test user data or something like this) ?

Reply:
since all data from the localstore is saved (not a specific key), this gives the advantage that if you change the name/edit/add/delete any of the keys to the localstore - do not have to edit the code in the tests

Reply:
i also dont understand why you need this
in tests you login either via non custodial using mnemonic/magiclink or by setting the bypass with TorusTestUser